src/main/java/org/junitpioneer/jupiter/NullEnum.java

```diff
@@ -16,5 +16,6 @@
  *
  * @see CartesianEnumSource#value()
  */
+@Deprecated
```

Comment:
The corresponding `@deprecated` Javadoc tag is missing:

https://sonarcloud.io/project/issues?id=junit-pioneer_junit-pioneer&open=AX2u626WwakKrVHzG7jF&pullRequest=556&resolved=false&types=CODE_SMELL